tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -57,15 +57,15 @@ static void to_display_format(const u8_t *src, size_t size, char *dst)
 struct spi_config spi_cfg_slow = {
 	.frequency = SLOW_FREQ,
 	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
-	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
+	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE | SPI_MODE_LOOP,
```

Comment:
First have a separate patch to modify this.
Second: don't do that always. On certain boards it's actually tested on hw, if loop mode is enabled, it will not work anymore etc...